ui/src/app/pages/analysis/notebook-resource-card.tsx

```diff
@@ -146,6 +160,8 @@ export const NotebookResourceCard = fp.flow(
   }
 
   copyNotebook(copyRequest: CopyRequest) {
+    AnalyticsTracker.Notebooks.Copy();
```

Comment:
Question why only in this case the AnalyticsTracker is mentioned inside the copyNotebook  function rather than in render

Reply:
It's because the render function is in a more generic `CopyModal` component. If I added the tracker there, I would have to add some logic so that the generic `CopyModal` knows what its parent component is.

I thought the cleaner way forward would be to add the tracker in this function since its what gets called `onClick`.